src/main/java/com/mg105/entities/Inventory.java

```diff
@@ -0,0 +1,61 @@
+package com.mg105.entities;
+import java.util.ArrayList;
+
+public class Inventory {
+    final static int inventoryLimit = 10;
+    private static ArrayList<Item> items = new ArrayList<>();
```

Comment:
Why are these static?

Reply:
I guess they do not really need to be static since we should only have one instance of the Inventory class but I just made them static just in case for some reason someone wanted to create another instance of inventory (it would still have the same items in the inventory).